### PR TITLE
Update TOC of first responder document

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

**Review by ndushay — APPROVED** [review]

**Review by jcoyne — APPROVED** [review]

File: infrateam_first_responder.md [hunk 1]
```diff
@@ -104,11 +111,6 @@ If on a Mac, you will get better results if you stay in the same "space" as the
 
 There are applications that need to be deployed separately (i.e., not using `sdr-deploy`).
```
Comment (ndushay) [inline infrateam_first_responder.md:113]: I think this sentence should also go??

**Review by ndushay — CHANGES_REQUESTED** [review]

File: infrateam_first_responder.md (hunk 1 above)
Comment (mjgiarlo) [reply infrateam_first_responder.md:113]: @ndushay I believe this sentence goes with the `Cloud Projects` that now immediately follows it. Should I strike that section and this sentence, given all our DLME and Sinopia work is done by previous steps (merging PRs) or CI? Or should we leave them both?

**Review by ndushay — APPROVED** [review]

File: infrateam_first_responder.md (hunk 1 above)
Comment (mjgiarlo) [reply infrateam_first_responder.md:113]: OK, then I guess let's leave them in there for now until we're confident we can stop thinking about them.

File: infrateam_first_responder.md [hunk 2]
```diff
@@ -10,11 +10,18 @@ The DLSS Infrastructure team is using a rotating role of "first responder." This
...
-      * [Important Exceptions](#important-exceptions)
-      * [Code that isn't a Ruby Application](#code-that-isnt-a-ruby-application)
+      * [1. Create a release tag](#1-create-a-release-tag)
+      * [2. Deploy to stage](#2-deploy-to-stage)
+      * [3. Run integration tests in stage](#3-run-integration-tests-in-stage)
+      * [4. Deploy to QA](#4-deploy-to-qa)
+      * [5. Deploy to prod](#5-deploy-to-prod)
+    * [Run infrastructure-integration-tests](#run-infrastructure-integration-tests)
+    * [Required Additional Deploys](#required-additional-deploys)
+      * [Ruby Projects with non-standard Capistrano targets](#ruby-projects-with-non-standard-capistrano-targets)
```
Comment (jmartin-sul) [inline infrateam_first_responder.md:20]:
```suggestion
```

i think this entry is no longer needed, since the section it links to was removed?

**Review by jmartin-sul — APPROVED** [review]
approved, with a suggested edit for one line of what seems like cruft removal

File: infrateam_first_responder.md (hunk 2 above)
Comment (mjgiarlo) [reply infrateam_first_responder.md:20]: @jmartin-sul Good eye! Committing this.

**Pull request merged**